src/routes/docs/quick-starts/go/+page.markdoc

```diff
@@ -215,6 +216,48 @@ func getTodos() {
 	var todos TodoList
 	todoResponse.Decode(&todos)
 
+	fmt.Println("Todos:")
+	for _, todo := range todos.Documents {
+		fmt.Printf("Title: %s\nDescription: %s\nIs Todo Complete: %t\n\n", todo.Title, todo.Description, todo.IsComplete)
+	}
+}
+
+func getCompletedTodos() {
+	// Use queries to filter completed todos with pagination
+	todoResponse, _ := appwriteDatabases.ListDocuments(
+		todoDatabase.Id,
+		todoCollection.Id,
+		appwriteDatabases.WithListDocumentsQueries([]string{
+			query.Equal("isComplete", true),
```

Comment:
Let's include the `"github.com/appwrite/sdk-for-go/query"` import in the  "Import Appwrite" section above.

Reply:
Sure, done